framework/test/jest/unit/specs/controller/helpers/validator/keywords/formatters.spec.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	stringToIpPortSet,
+	stringToIpList,
+	stringToDelegateList,
+} = require('../../../../../../../../src/controller/validator/keywords/formatters');
+
+describe('formatters', () => {
+	describe('stringToIpPortSet', () => {
+		it('should return empty array if given argument is not string', async () => {
```

Comment:
do we need `async` here? same applies to all other cases below.

Reply:
Theoretically no need. There were a es-lint check earlier no-sync for mocha tests, so I had habit to have every test async.